llm_ollama.py

```diff
@@ -191,6 +205,28 @@ def build_messages(self, prompt, conversation):
         return messages
 
 
+class OllamaEmbed(llm.EmbeddingModel):
+    supports_text = True
+    supports_binary = False
+    batch_size = 8
```

Comment:
the text and binary are because the embed endpoint only accepts text input
the batch size is arbitrary, llm can overwrite the value when calling, but if no value is given it defaults to this value of 8